app/helpers/katello/concerns/foreman_docker/container_steps_helper_extensions.rb

```diff
@@ -0,0 +1,78 @@
+#
+# Copyright 2014 Red Hat, Inc.
+#
+# This software is licensed to you under the GNU General Public
+# License as published by the Free Software Foundation; either version
+# 2 of the License (GPLv2) or (at your option) any later version.
+# There is NO WARRANTY for this software, express or implied,
+# including the implied warranties of MERCHANTABILITY,
+# NON-INFRINGEMENT, or FITNESS FOR A PARTICULAR PURPOSE. You should
+# have received a copy of GPLv2 along with this software; if not, see
+# http://www.gnu.org/licenses/old-licenses/gpl-2.0.txt.
+
+module Katello
+  module Concerns
+    module ForemanDocker
+      module ContainerStepsHelperExtensions extend ActiveSupport::Concern
+        def katello_partial(f)
```

Comment:
Perhaps add another argument ```path``` and let it render ```:partial => path```, if you want to make a generic katello_partial renderer. ```katello_partial``` sounds so generic it could be any partial.

I like it because it makes foreman-docker code that uses katello partials very explicit, just keep in mind ```<%= render(:partial => "/foreman_docker/containers/steps/katello_container", :locals => {:f => f}) %>
``` would work just as well in foreman-docker directly, in case you don't want to maintain this method.